ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -617,6 +649,9 @@ S3Terminate(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session)
         S3FileSystemTerminate(&fs->fileSystem, session);
     }
 
+    /* Log collected statistics on termination. */
+    S3ShowStatistics(s3);
```

Comment:
We should make dumping statistics conditional on the verbosity level. Avoid showing statistics at the default level. I suspect our python test will not like messages printed into the stdout, and will treat it as an error.

Reply:
The python test doesn't seem not seem to treat unexpected `std::cout` as error. I'm planning to change statistics to be conditional on the verbosity level in the next logging ticket.